runtime/bindings/python/src/openvino/ie_api.py

```diff
@@ -104,11 +110,6 @@ def __new__(cls, tensor_desc: TensorDesc, arr: np.ndarray = None):  # type: igno
         else:
             raise AttributeError(f"Unsupported precision {precision} for Blob")
 
-# flake8: noqa: D102
-def blob_from_file(path_to_bin_file: str) -> BlobWrapper:
```

Comment:
If you remove all blob related funcs, please remove `BlobWrapper` and `Blob` pybind related files as well.

Reply:
I wanted to remove all of them but ie_process_info.cpp in core still uses Blob so I removed all the functions that are not used anywhere

Reply:
you may remove ```ie_preprocess_info```, it's useless now because it comes from old API

Reply:
revert the removal of the Blob components in this PR